rust-runtime/aws-smithy-http-server-python/Cargo.toml

```diff
@@ -22,6 +22,7 @@ bytes = "1.2"
 futures = "0.3"
 http = "0.2"
 hyper = { version = "0.14.20", features = ["server", "http1", "http2", "tcp", "stream"] }
+lambda_http = "0.6.0"
```

Comment:
Not a blocker for this PR, but I would probably create an issue to feature-gate lambda in the future so that we do not impose this dependency on all customers. I am not even sure if its worth it thou.. Thoughts?

Reply:
Yes good point. But I'm not sure if we currently support to generate SDKs with specific features on dependencies?